### PR TITLE
Actualize features list in README

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,14 +19,19 @@ This package provides data validation capabilities.
 
 ## Features
 
-- Can be used with any object.
+- Can be used with any data, including arrays and objects. It's possible to create custom data sets too.
```
Comment (vjik) [inline README.md:22]:
```suggestion
- Validate any data: arrays, objects, scalar values, etc.
- Supports custom data sets.
```

File: README.md [hunk 2]
```diff
@@ -19,14 +19,19 @@ This package provides data validation capabilities.
 
 ## Features
 
-- Can be used with any object.
+- Can be used with any data, including arrays and objects. It's possible to create custom data sets too.  
+- Handles data relations.
```
Comment (vjik) [inline README.md:23]:
```suggestion
- Handles nested data structures (one-to-one and one-to-many).
```

File: README.md (hunk 1 above)
Comment (samdark) [reply README.md:22]: Validate but Support**s**.

I'd suggest `validates`.

File: README.md [hunk 3]
```diff
@@ -19,14 +19,20 @@ This package provides data validation capabilities.
 
 ## Features
 
-- Can be used with any object.
+- Validate any data: arrays, objects, scalar values, etc.
```
Comment (arogachev) [inline README.md:22]:
```suggestion
- Validates any data: arrays, objects, scalar values, etc.
```

**Pull request merged**